packages/editor-ui/src/components/WorkflowActivator.vue

```diff
@@ -54,6 +55,9 @@ export default mixins(
 				};
 			},
 			computed: {
+				...mapGetters({
+					isDirty: "getStateIsDirty",
```

Comment:
names that start with `is` are usually expected to be a function. Would recommend to rather choose a name that is understood as a bool
```suggestion
					dirtyState: "getStateIsDirty",
```